Sources/XcodeGenKit/SourceGenerator.swift

```diff
@@ -85,6 +85,7 @@ class SourceGenerator {
         let fileReference = fileReference ?? fileReferencesByPath[path.string.lowercased()]!
         var settings: [String: Any] = [:]
         var attributes: [String] = targetSource.attributes
+        let resourceTags: [String] = targetSource.resourceTags
```

Comment:
This `resourceTags` property is only used once, so let's just remove it and access `targetSource.resourceTags` inline down below

Reply:
91c45bf